src/lib/MetadataAPI.js

```diff
@@ -0,0 +1,56 @@
+import api from './api';
+import { getHeaders } from './util';
+
+// Map of extensions to global metadata template
+const EXTENSIONS_TO_TEMPLATES = {
```

Comment:
Extensions could load multiple templates, in theory. Should the template(s) needed just be passed in by the calling code, rather than living in a map within the API layer?

Reply:
Should we also pass in the scope in case we want to access templates at other levels in the future?